libraries/Zigbee/examples/Zigbee_On_Off_Light/Zigbee_On_Off_Light.ino

```diff
@@ -0,0 +1,86 @@
+// Copyright 2024 Espressif Systems (Shanghai) PTE LTD
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @brief This example demonstrates simple Zigbee light bulb.
+ *
+ * The example demonstrates how to use Zigbee library to create a end device light bulb.
+ * The light bulb is a Zigbee end device, which is controlled by a Zigbee coordinator.
+ *
+ * Proper Zigbee mode must be selected in Tools->Zigbee mode
+ * and also the correct partition scheme must be selected in Tools->Partition Scheme.
+ *
+ * Please check the README.md for instructions and more detailed description.
+ * 
+ * Created by Jan Procházka (https://github.com/P-R-O-C-H-Y/)
+ */
+
+#ifndef ZIGBEE_MODE_ED
+#error "Zigbee end device mode is not selected in Tools->Zigbee mode"
+#endif
+
+#include "ZigbeeCore.h"
+#include "ep/ZigbeeLight.h"
+
+#define LED_PIN RGB_BUILTIN
+#define BUTTON_PIN 9  // ESP32-C6/H2 Boot button
+#define ZIGBEE_LIGHT_ENDPOINT 10
+
+ZigbeeLight zbLight = ZigbeeLight(ZIGBEE_LIGHT_ENDPOINT);
+
+/********************* RGB LED functions **************************/
+void setLED(bool value) {
+  rgbLedWrite(LED_PIN, 255 * value, 255 * value, 255 * value);
```

Comment:
you should use `digitalWrite` here (which works with LED_BUILTIN)

Reply:
Would it make sense to have both with one commented out?
All of the boards I have so far use Neopixel not a single LED.

Reply:
If board is defined properly, `digitalWrite` will work with neopixel as well. The point is to show that it's an on/off thing, rather than changing color.

Reply:
@P-R-O-C-H-Y please point to proper neopixel as board led config example

Reply:
Done + left a comment that `rbgLedWrite()` will be used under the hood if `LED_PIN == RGB_BUILTIN`